Simple Inventory Management System/Inventory.cs

```diff
@@ -28,5 +28,40 @@ public void DisplayProducts()
             }
         }
 
+        public void EditProduct(string productName)
+        {
+            Product product = products.Find(p => p.Name == productName);
+            if (product != null)
+            {
+                Console.WriteLine($"Product found: {product.Name} - Price: ${product.Price} - Quantity: {product.Quantity}");
+                Console.WriteLine("Enter new details:");
+
+                Console.Write("New name (leave blank to keep unchanged): ");
+                string newName = Console.ReadLine();
+                if (!string.IsNullOrEmpty(newName))
+                {
+                    product.Name = newName;
+                }
+
+                Console.Write("New price : ");
+                string newPriceStr = Console.ReadLine();
```

Comment:
For the new price you ask for the input without specifying that the user can leave it blank to keep it unchanged.